rep-0003.rst

```diff
@@ -343,6 +343,53 @@ Build System Support:
 
 - Same as Indigo
 
+Noetic Ninjemys (May 2020 - May 2025)
+--------------------------------------
+Required Support for:
+
+- Ubuntu F----- (20.04)
```

Comment:
```suggestion
- Ubuntu Focal Fossa (20.04)
```

Reply:
I thought the dashes covered up some sort of profanity ;)